include/fuzzuf/algorithms/afl/templates/afl_mutation_hierarflow_routines.hpp

```diff
@@ -329,7 +329,7 @@ AFLMutCalleeRef<State> UserDictInsertTemplate<State>::operator()(
     u64 orig_hit_cnt = state.queued_paths + state.unique_crashes;
 
     std::unique_ptr<u8[]> ex_tmp(new u8[mutator.GetLen() + option::GetMaxDictFile(state)]);
-    for (u32 i=0; i <= mutator.GetLen(); i++) {
```

Comment:
I think this is not a mistake but an intended range. Consider the case where we want to insert `"B"` into somewhere in `"AA"`: there are three(=len("AA")+1) patterns, `"BAA"` `"ABA"` `"AAB"`.

Reply:
Replaced by more appropriate fix that avoid inserting out of range value on last cycle instead to change loop condition.